lib/fs.js

```diff
@@ -1886,6 +1886,7 @@ function WriteStream(path, options) {
   this.mode = options.mode === undefined ? 0o666 : options.mode;
 
   this.start = options.start;
+  this.autoClose = options.autoClose === undefined ? true : options.autoClose;
```

Comment:
Can you add a `!!` on the last `options.autoClose` to guarantee we have a Boolean.